site/_posts/2016-02-08-jekyll-3-0-3-released.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: news_item
+title: 'Jekyll 3.0.3 Released'
+date: 2016-02-08 10:39:08 -0800
+author: parkr
+version: 3.0.3
+categories: [release]
+---
+
+[GitHub Pages upgraded to Jekyll 3.0.2][1] last week and there has been a
```

Comment:
Fine as is, but:

1. *some* users had *some* bugs feels a bit weasily
2. Joyous reception? Show, don't tell. If we can't show, I'd cut it.
3. What if we make this a positive, instead of a negative? It's not that we :ship:'d with a bunch of bugs, but the cool thing about having a test bed of 1M+ sites is that you can quickly and easily surface unique bugs and edge cases.